src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -396,7 +396,7 @@ def recover_hsm(cmd, client, hsm_name, resource_group_name, location, no_wait=Fa
         resource=cmd.cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
 
     parameters = ManagedHsm(location=location,
-                            sku=ManagedHsmSku(name='Standard_B1'),
+                            sku=ManagedHsmSku(name='Standard_B1', family='B'),
```

Comment:
can user specify the sku?

Reply:
@yungezz No, this is a fixed value, for now, users don't have to know this.